next/components/nav/AuthButton.tsx

```diff
@@ -6,14 +6,14 @@ export default function AuthButton() {
 
     if (session) {
         return (
-            <HoverBoldButton text="Logout" dataLabel="Logout" onClick={() => signOut()} />
+            <HoverBoldButton className="text-left" text="Logout" dataLabel="Logout" onClick={() => signOut()} />
```

Comment:
Curious, the other buttons in the navbar, are they also text-left or are they text-center? I feel like it would make more sense for all of them to be text-center

Reply:
> Curious, the other buttons in the navbar, are they also text-left or are they text-center? I feel like it would make more sense for all of them to be text-center

They are text-left. I made sure the login button was consistent with the other buttons that were already there.

Reply:
OH that's my bad. I was thinking about the wrong thing XD